net/eoip/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2006-2012 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=eoip
+PKG_VERSION:=0.5
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Bogdik <bogdikxxx@gmail.com>
+PKG_LICENSE:=GPL-2.0
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://github.com/bogdik/openwrt-linux-eoip/releases/download/0.6.1/
+PKG_MD5SUM:=ab6afb3eb806aee7e6dbfad2bf74de9b
+
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/eoip/Default
+  TITLE:=Mikrotik-compatible userspace EoIP tunnel
+  URL:=http://code.google.com/p/linux-eoip/
+endef
+
+define Package/eoip
+  $(call Package/eoip/Default)
```

Comment:
As you only have a normal Package/eoip package, there is no need to a separate rather empty Package/eoip/Default definition. You should combine its contents here.